helm-charts/konk-service/values.yaml

```diff
@@ -48,3 +48,9 @@ service:
   name: test
 
 crds: null
+
+group:
+
+version:
+
+tlsVerify: false
```

Comment:
We should default this to `true`.

Reply:
You're really confusing me now. Is this feature verifying or skipping verification? If the variable is named opposite of what it does, the name should be changed.

Reply:
You're right. Will rename it to `tlsVerifySkip`.

Reply:
It would be least confusing to name it exactly after the k8s field: `insecureSkipTLSVerify`
https://github.com/infobloxopen/konk/blob/4ac9771d63fcad89759c33758111042a5244b962/helm-charts/konk-service/templates/configmap.yaml#L59